Makefile

```diff
@@ -17,6 +17,7 @@ requirements: requirements.txt dev-requirements.txt
 
 .PHONY: setup
 setup: install-piptools ## Install requirements
+	curl -s https://raw.githubusercontent.com/lyft/flytectl/master/install.sh | bash
```

Comment:
use Github action rather than shell script https://github.com/unionai/flytectl-setup-action

```
  - uses: actions/checkout@v2
  - uses: unionai/flytectl-setup-action@v0.0.1
    with:
      version: '0.1.8' 
```